src/sentry/monitors/logic/mark_failed.py

```diff
@@ -117,18 +117,16 @@ def mark_failed_threshold(failed_checkin: MonitorCheckIn, failure_issue_threshol
         # use .values() to speed up query
         previous_checkins = list(
             reversed(
-                MonitorCheckIn.objects.filter(monitor_environment=monitor_env)
+                MonitorCheckIn.objects.filter(
+                    monitor_environment=monitor_env, date_added__lte=failed_checkin.date_added
```

Comment:
I don't think we should leave the `date_added__lte` here though right, since check-ins that fail way back won't correctly look at the more recent failures right? It might work OK for the edge case we're trying to fix for the customer who is just having successive time outs and failures, but it may fail in other scenarios.

For now it's probably simpler to just drop the `date_added__lte`

Reply:
Okay chatted with @rjo100 again about this, and just going to cover it in https://github.com/getsentry/sentry/issues/60123 as well